lib/rspec/core/bisect/example_minimizer.rb

```diff
@@ -43,8 +86,27 @@ def repro_command_for_currently_needed_ids
           "(Not yet enough information to provide any repro command)"
         end
 
+        # @private
+        # Convenience class for describing a subset of the candidate examples
+        ExampleRange = Struct.new(:start, :finish) do
+          def description
+            if start == finish
+              "example #{start}"
+            else
+              "examples #{[start, finish].join("-")}"
```

Comment:
This seems simpler as:

``` ruby
"examples #{start}-#{finish}"
```

...which should also be a bit faster -- it's fewer object allocations (no unnecessary array allocation or string allocation for `"-"`) and fewer method calls (no `join` call).